app/src/main/java/org/sil/gatherwords/SessionActivity.java

```diff
@@ -86,7 +86,7 @@ protected void onCreate(Bundle savedInstanceState) {
             dateField.setText(dateSDF.format(date));
             timeField.setText(timeSDF.format(date));
             String timeZoneString = timeZoneSDF.format(date);
-            timeZoneField.setText(timeZoneString.substring(3, timeZoneString.length()));
+            timeZoneField.setText(timeZoneString.substring(3)); // GMT+X:00 >>> +X:00
```

Comment:
Do we know time zone will always be GMT?  Not all time zones are exactly three characters.

Reply:
It is offset from GMT, so it will always start with GMT